crates/test-ui/ui/msg_send_underspecified_error.stderr

```diff
@@ -1,89 +1,5 @@
 error[E0282]: type annotations needed
  --> ui/msg_send_underspecified_error.rs
   |
-  |     let _: Result<Id<NSString>, Id<_>> = unsafe { msg_send_id![obj, d: _] };
-  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type
-
-error[E0283]: type annotations needed
- --> ui/msg_send_underspecified_error.rs
-  |
-  |     let _: Result<(), _> = unsafe { msg_send![obj, a: _] };
-  |                                     ^^^^^^^^^^^^^^^^^^^^ cannot infer type for raw pointer `*mut _`
-  |
-  = note: multiple `impl`s satisfying `*mut _: RefEncode` found in the `objc2` crate:
-          - impl RefEncode for *mut c_void;
-          - impl<T> RefEncode for *mut T
-            where T: RefEncode, T: ?Sized;
-  = note: required for `*mut *mut _` to implement `Encode`
-note: required by a bound in `send_message_error`
- --> $WORKSPACE/crates/objc2/src/__macro_helpers/msg_send.rs
-  |
-  |     unsafe fn send_message_error<A, E>(self, sel: Sel, args: A) -> Result<(), Id<E>>
-  |               ------------------ required by a bound in this associated function
-  |     where
-  |         *mut *mut E: Encode,
-  |                      ^^^^^^ required by this bound in `MsgSend::send_message_error`
-  = note: this error originates in the macro `$crate::__msg_send_helper` which comes from the expansion of the macro `msg_send` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
It may seem unfortunate that we're now missing this information, but in reality it's not actually helpful for the user.